examples/module/public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>WRLD Map</title>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.1/leaflet.css" rel="stylesheet" />
```

Comment:
We can import that easily with a native CSS call

```css
@import url("https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.1/leaflet.css");
```

The problem is that it would cause the same CSS being loaded twice in the applications loading wrld.css

While it shouldn't be a breaking change, it will introduce a double loading. We could try and detect that via the JS and show a warning, though.

What do you think?

Reply:
I was looking at this now, and `wrld.css` (<https://github.com/wrld3d/eegeo-js-common/blob/31301d8fc83d2bf53be1506adcd4278bc998e026/wrld-css/scss/main.scss#L0-L1>) has a lot of extra stuff going on (mostly for the widgets). 

- wrld.css is 9.4 kB
- leaflet.css is 3.4 kB

Importing wrld.css when they just need leaflet style would mean another 9.4 kB of useless stuff. On the other hand, they would have everything ready if they want to import the widgets.